pkg/proc/proc_test.go

```diff
@@ -3697,3 +3697,20 @@ func TestIssue951(t *testing.T) {
 		}
 	})
 }
+	
+func TestDWZCompression(t *testing.T) {
+	// If dwz is not available in the system, just ignore this test
+	if _, err := exec.LookPath("dwz"); err != nil {
+		return
```

Comment:
We are moving towards using Skip instead of returning, so this should be `t.Skip("dwz not installed")`